### DISCUSSION TITLE
Hidden comments

### DISCUSSION THREAD

**Discussion by TPS** [community/community#4124] [Mobile]
Via the website interfaces, if 1 has sufficient permissions (@ least collaborator, I believe), 1's able to hide (not just delete) comments, which then should appear something like <kbd>This comment was hidden.  Unhide?</kbd>.

This is _still_ not available via the mobile app:
- [x] the ability to hide comments isn't available (though deleting _is_ 😕)
- [ ] comments that _are_ marked hidden still appear by default

**Comment by aconsuegra** [comment]
Thank you so much for the request @TPS! 

We are tracking being able to hide comments internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it! 

> comments that are marked hidden still appear by default
And this has been added recently to comments in Discussions, but we still need to add it to other places of the app.

**Comment by viceice** [comment]
I also like to be able to hide comments from mobile app

**Comment by yazabes** [comment]
I need the password to acces my email bryanperz@hotmail.com 
To access Instagram
